controller/dev-requirements.txt

```diff
@@ -0,0 +1 @@
+micropy-cli
```

Comment:
What's this file for?

Reply:
This file was auto-generated by micropy-cli, which I am using to manage device-specific dependencies. It looks like micropy wants to differentiate between device packages and development environment packages. I'll try moving it's contents to `requirements.txt` and see if that helps. That might be why the build is not working on your machine, per your comment below. Using micropy-cli might require us to change the configuration of `controller/`, but I'll see what I can do.